test/e2e/resources/full-chart-install.yaml

```diff
@@ -17,7 +17,7 @@ spec:
         caBundle: Cg==
         service:
           name: capi-operator-webhook-service
-          namespace: 'capi-operator-system'
+          namespace: 'default'
```

Comment:
any reason why is this needed for generating full chart?

Reply:
This is the default state of the chart.